core/templates/pages/learner-group-pages/edit-group/learner-group-learner-specific-progress.component.html

```diff
@@ -95,13 +95,16 @@
                       <span>Lesson {{idx+1}}: {{chapterName}}</span>
```

Comment:
I18N key for `Lessons`. Please make sure there is no raw text on the entire page since its learner facing.

Reply:
done